doc/engineering-reference/src/surface-heat-balance-manager-processes/conduction-through-the-walls.tex

```diff
@@ -61,7 +61,27 @@ \subsection{Calculation of Conduction Transfer Functions}\label{calculation-of-c
+Seem (1987) summarizes the steps required to obtain the output vector (y) from the A, B, C, and D matrices.~ Briefly, the solution for the first order differential system of equations with constant coefficients shown above is:
+
+\begin{equation}
+x_{t + \delta} = e^{A \delta} x_t + \int\limits_t^{t + \delta } {e^{A (t + \delta - \tau)} B u(\tau) d \tau}
+\end{equation}
+
+where t is a particular point in time, \(\delta\) is the time step, and \(e^{A \delta}\) is the exponential matrix which is calculated using a power series:
+
+\begin{equation}
+e^{A \delta} = I + A \delta + \frac{A^2 \delta ^ 2}{2!} + \frac{A^3 \delta ^ 3}{3!} + \ldots{} + \frac{A^L \delta ^ L}{L!} + \ldots{}
+\end{equation}
+
+where L is the term in the sequence that defines an adequate representation of the exponential matrix.  In other words, terms beyond L are not needed because they will not supply any significant additional value to the overall exponential matrix.  So, the calculation of the exponential matrix can be terminated after the Lth term.  Seem (1987) also defines what L should be in Appendix A of his dissertation.  It is based on an equation that relates L to the matrix row norm:
+
+\begin{equation}
+L = minimum~of~[3 ||A \delta||_\infty + 6]~or~100
+\end{equation}
+
+where \(||A \delta||_\infty\) is the matrix row norm.  Seem, as part of his algorithm, indicates that the matrix row norm should be normalized so that it is less than unity by finding the power of 2 that is larger than the matrix row norm and dividing all elements of the A matrix by that term.  This is used later as part of the "scaling and squaring" approach to calculate the exponential matrix.  This scaling would also affect L meaning that if the matrix row norm has been scaled so that it is never greater than 1 then L would never go higher than 9.  This seems somewhat unsual (especially given the equation for L has a limit of 100) and when Seem's approach was first implemented in EnergyPlus, the matrix row norm was NOT scaled when it was used to calculate L.  So, while the actual implementation in EnergyPlus deviates slightly from what was recommended by Seem, this approach can perhaps be seen to reflect the improvements in computer technology since that work was published or simply that there appears to be a potential incongruency in the method that Seem suggested for that one detail.  In addition, to avoid excessive terms in the exponential matrix, the terms that are added in the exponential matrix calculation in EnergyPlus are limited so that if a new term is smaller than will impact the variable based on the precision of the variable it is then ignored.  To see the implementation of these details in EnergyPlus, consult both Appendix A of Seem's dissertation and the routine calculateExponentialMatrix in the code.
```

Comment:
A suggested edit:
~This seems somewhat unsual (especially given the equation for L has a limit of 100) and when Seem's approach was first implemented~ In EnergyPlus, the matrix row norm ~was~ *is* not scaled when ~it was~ used to calculate L.  ~So, while the actual implementation in EnergyPlus deviates slightly from what was recommended by Seem, this approach can perhaps be seen to reflect the improvements in computer technology since that work was published or simply that there appears to be a potential incongruency in the method that Seem suggested for that one detail.~ *This can result in additional terms which in general will make the calculation slightly more accurate.* ~In addition,~ To avoid excessive terms in the exponential matrix,  . . .

Reply:
I'll make these changes as requested also.  Thanks for reviewing all this, @mjwitte and @mitchute!